plugins/embed-optimizer/hooks.php

```diff
@@ -181,8 +181,9 @@ function embed_optimizer_update_markup( WP_HTML_Tag_Processor $html_processor, b
 		'script' => 'embed_optimizer_script',
 		'iframe' => 'embed_optimizer_iframe',
 	);
-	$trigger_error  = static function ( string $message ): void {
-		wp_trigger_error( __FUNCTION__, esc_html( $message ) );
+	$function_name  = __FUNCTION__;
+	$trigger_error  = static function ( string $message ) use ( &$function_name ): void {
+		wp_trigger_error( $function_name, esc_html( $message ) );
```

Comment:
Good catch. This was wrong!

Reply:
right, PHPSTan caught that - `__FUNCTION__` is not the same inside a closure.